packages/grafana-ui/src/components/Alert/Alert.tsx

```diff
@@ -1,54 +1,64 @@
 import React, { FC, ReactNode } from 'react';
-import classNames from 'classnames';
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
 import { Icon } from '../Icon/Icon';
 import { IconName } from '../../types/icon';
 
 export type AlertVariant = 'success' | 'warning' | 'error' | 'info';
 
-interface AlertProps {
+export interface Props {
   title: string;
-  buttonText?: string;
-  onButtonClick?: (event: React.MouseEvent) => void;
   onRemove?: (event: React.MouseEvent) => void;
   severity?: AlertVariant;
   children?: ReactNode;
+  removeButtonContent?: ReactNode | string;
+  /** Deprecated. Use onRemove instead */
```

Comment:
I think adding `@deprecate` annotation might be helpful here from docs generation perspective

Reply:
Ah, great, didn't know it worked here too.